src/NzbDrone.Update.Test/ProgramFixture.cs

```diff
@@ -32,8 +32,9 @@ public void should_throw_if_first_arg_isnt_an_int(string arg1, string arg2)
         }
 
         [Test]
-        public void should_call_update_with_corret_path()
+        public void should_call_update_with_correct_path_windows()
```

Comment:
The test fails in `UpdateApp.cs` due to Start being called with two arguments. When parsing args there is a check if the OS is not Windows (https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Update/UpdateApp.cs#L74) and it requires either 1 or at least 3 arguments. Otherwise an `IndexOutOfRangeException` is thrown.